src/super_gradients/evaluate_from_recipe.py

```diff
@@ -0,0 +1,44 @@
+"""
+Evaluate a SuperGradient's recipes.
+
+Use this script if:
+    - You want to evaluate a pretrained model from model zoo
+    - You want to evaluate a checkpoint from one of your previous experiment, but with different validation parameters
+        such as dataset params or metrics for instance
+
+Don't use this script if:
+    - You want to train and evaluate a model (use examples/train_from_recipe_example)
+    - You want to evaluate a checkpoint from one of your previous experiment, using the same parameters as used during the
+        training of the experiment (use examples/evaluate_checkpoint_example)
+
+Note:
+    This script does NOT run TRAINING, so make sure in the recipe that you load a PRETRAINED MODEL
+    either from one of your checkpoint or from a pretrained model.
+
+General use: python -m super_gradients.evaluate_from_recipe --config-name="DESIRED_RECIPE".
+-> Evaluate the latest checkpoint according to parameters set in "DESIRED_RECIPE"
+
+You can specify which checkpoint you want to evaluate by overriding training_hyperparams.ckpt_name as in the following example:
+python -m super_gradients.evaluate_from_recipe --config-name="DESIRED_RECIPE" training_hyperparams.ckpt_name=average_model.pth
+-> Evaluate the checkpoint 'average_model.pth' according to parameters set in "DESIRED_RECIPE"
+
+For recipe's specific instructions and details refer to the recipe's configuration file in the recipes directory.
+"""
+from omegaconf import DictConfig
+import hydra
+
+from super_gradients import Trainer, init_trainer
+
+
+@hydra.main(config_path="recipes", version_base="1.2")
```

Comment:
`@hydra.main(config_path=package_ressources("super_gradients.recipes", ""), ...)` crashes the code after moving to the root of sg
```
Primary config module 'super_gradients..home.louis.dupont.PycharmProjects.super-gradients.src.super_gradients.recipes' not found.
Check that it's correct and contains an __init__.py file
```

This can be fixed by just using `@hydra.main(config_path="recipe", ...)` .
I don't see a reason not to use `config_path="recipes"` here anyway, do you have any thoughts about that?

Reply:
IMO, should be fine to use config_path="recipes". Can't think of cases when it can break.